Jenkinsfile

```diff
@@ -51,7 +51,10 @@ pipeline {
         sh './doc/indent'
         sh 'git diff | tee changes-astyle.diff'
         archiveArtifacts artifacts: 'changes-astyle.diff', fingerprint: true
-        sh 'git diff --exit-code --name-only'
+        sh '''
+	  git diff --exit-code --name-only || \
+	  { echo "Please check indentation, see artifacts in the top right corner!"; exit 1; }
```

Comment:
I had just seen this as well when looking at a PR. Why not just run `git diff` again here to show the actual diffs? I know you already do this above and output the result, but it's awkward to find because it shows up in a green tab. The failure output should show up in a red tab like it does here, so if you ran `git diff` instead of/in addition to the `echo` here, you can remove lines 52/53

Reply:
52/53 has the advantage that you can download the .diff file directly and apply the diff.

Reply:
@gassmoeller what do you think. Should we just show the whole diff here?

Reply:
You could do what you do in lines 52/53 only in here where you know that things have failed.

Reply:
I can not conditionally archive the diff, but I think this does what you want now.